src/models/DefaultData.js

```diff
@@ -119,27 +119,22 @@ class DefaultData {
   }
 
   /**
-   * Rename a component ID and update all its occurence inside link/References attribute type.
-   * @param {string} oldId - Old ID of component.
-   * @param {string} newId - New ID of component.
+   * Get component by configuration any kind of key.
```

Comment:
Rework on documentation, configuration key is used in case of get component by the key used in the configuration file that saved the component position, and this method is only used when we get position from configuration. So adapt the documentation.

Reply:
done